### DISCUSSION TITLE
Sincronizzazione

### DISCUSSION THREAD

**Discussion by Cristofer27** [Chia-Network/chia-blockchain#16772] [Support]
ciao,sono giorni che la piattaforma chia resta in Sincronizzazione e non passa in connessione come posso fare?grazie

**Comment by BrandtH22** [comment]
Ciao @Cristofer27, tieni presente che possiamo fornire risposte più tempestive e approfondite nel nostro server discord (https://discord.gg/chia). 

Possono essere necessari giorni per sincronizzare l'intera catena dalla genesi. Se vedi che l'altezza corrente aumenta, stai ancora sincronizzando correttamente. 
Per velocizzare il processo di sincronizzazione, puoi utilizzare l'istantanea ufficiale da qui (https://www.chia.net/downloads/#database-checkpoint).

Passaggi per l'utilizzo del torrent: 
```
1. Scarica il file torrent da https://www.chia.net/downloads/#database-checkpoint 
2. Usa un client torrent per scaricare l'intero db 
3. Decomprimere/riassemblare il file torrent che è stato scaricato (su Windows ho usato 7zip, Mac e Linux hanno strumenti integrati che funzionano per questo) 
4. Spostare il db nella cartella corretta (~\.chia\mainnet\db\) e aggiornare il nome in "blockchain_v2_mainnet.sqlite" (rimuovendo le informazioni sulla data nel nome) 
5. Avvia chia e attendi un po' (l'altezza per l'hash e i file peer devono essere compilati in modo che questo possa richiedere fino a 30 minuti)
```

Questo messaggio è stato tradotto in parte utilizzando un software di traduzione

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
